packages/insured-bridge-relayer/src/Relayer.ts

```diff
@@ -727,66 +652,159 @@ export class Relayer {
-          at: "InsuredBridgeRelayer#Relayer",
+          at: "AcrossRelayer#Relayer",
           message: "Not relaying potentially unprofitable deposit, or insufficient balance 😖",
           realizedLpFeePct: realizedLpFeePct.toString(),
           relayState: relayableDeposit.status,
           hasInstantRelayer,
           relayableDeposit,
         });
-        break;
+        return { transaction: null, message: "", mrkdwn: "" };
       case RelaySubmitType.Slow:
         this.logger.debug({
-          at: "InsuredBridgeRelayer#Relayer",
+          at: "AcrossRelayer#Relayer",
           message: "Slow relaying deposit",
           realizedLpFeePct: realizedLpFeePct.toString(),
           relayableDeposit,
         });
-        await this.slowRelay(relayableDeposit.deposit, realizedLpFeePct);
-        break;
+        return {
+          transaction: this._generateSlowRelayTx(relayableDeposit.deposit, realizedLpFeePct),
+          message: "Slow Relay executed  🐌",
+          mrkdwn,
+        };
 
       case RelaySubmitType.SpeedUp:
         this.logger.debug({
-          at: "InsuredBridgeRelayer#Relayer",
+          at: "AcrossRelayer#Relayer",
           message: "Speeding up existing relayed deposit",
           realizedLpFeePct: realizedLpFeePct.toString(),
           relayableDeposit,
         });
-        if (pendingRelay === undefined)
+        if (pendingRelay === undefined) {
           // The `pendingRelay` should never be undefined if shouldRelay returns SpeedUp, but we have to catch the
           // undefined type that is returned by the L1 client method.
-          this.logger.error({ at: "InsuredBridgeRelayer#Relayer", message: "speedUpRelay: undefined relay" });
-        else await this.speedUpRelay(relayableDeposit.deposit, pendingRelay);
-        break;
+          this.logger.error({ at: "AcrossRelayer#Relayer", message: "speedUpRelay: undefined relay" });
+          return { transaction: null, message: "", mrkdwn: "" };
+        } else
+          return {
+            transaction: this._generateSpeedUpRelayTx(relayableDeposit.deposit, pendingRelay),
+            message: "Slow relay sped up 🏇",
+            mrkdwn,
+          };
+
       case RelaySubmitType.Instant:
         this.logger.debug({
-          at: "InsuredBridgeRelayer#Relayer",
+          at: "AcrossRelayer#Relayer",
           message: "Instant relaying deposit",
           realizedLpFeePct: realizedLpFeePct.toString(),
           relayableDeposit,
         });
-        await this.instantRelay(relayableDeposit.deposit, realizedLpFeePct);
-        break;
+        return {
+          transaction: this._generateInstantRelayTx(relayableDeposit.deposit, realizedLpFeePct),
+          message: "Relay instantly sent 🚀",
+          mrkdwn,
+        };
     }
   }
 
+  private _generateMarkdownForRelay(deposit: Deposit, realizedLpFeePct: BN) {
+    return (
+      "Relayed " +
+      this._generateMrkdwnDepositIdNetworkSizeFromTo(deposit) +
```

Comment:
As mentioned IRL I think its important to at least include the deposit and relay ID for easier tracking

Reply:
I suppose you could find those easily in the etherscan link of the `tx`

Reply:
Deposit ID is included already. see `_generateMrkdwnDepositIdNetworkSizeFromTo`

Relay ID is not know until we get the events. I can refactor this to capture this data but would like to leave that for a subsequent PR.